src/renderers/table.test.ts

```diff
@@ -406,4 +447,45 @@ describe('given a table entry', () => {
       );
     });
   });
+
+  describe('with prefixCellValues option set', () => {
+    test.each([[undefined], [true]])(
+      'applies prefix to cell values when prefixCellValues = %s',
+      (prefixCellValues) => {
+        expect(
+          tsMarkdown([
+            {
+              blockquote: {
+                table: {
+                  columns: ['Col1', 'Col2'],
+                  rows: [['Row 1-Left', 'Row 1-Right']],
+                },
+                prefixCellValues,
+              },
+            },
+          ])
+        ).toBe(`> | Col1         | Col2          |
+> | ------------ | ------------- |
+> | > Row 1-Left | > Row 1-Right |`);
```

Comment:
This test illustrates how the "prefix" is prepended to each individual cell, which is what `prefixCellValues` is intended to disable.